helm/cluster-cloud-director/values.yaml

```diff
@@ -92,6 +92,10 @@ nodePools: {}
   # worker:
   #   class: default  # Node classes name (defined above)
   #   replicas: 2
+ntp:
+  servers: 
+  - ""
+  enabled: false
```

Comment:
Maybe we don't need the `enabled` switch and only add the ntp config to the chart if `servers` is populated.